src/components/SearchRoot.js

```diff
@@ -92,10 +93,12 @@ export let SearchRoot = DDContext(
         ...(tree.tree && ContextureClientBridge(types, tree)),
       }),
     }),
-    ({ tree, state, types = {} }) => (
+    ({ tree, state, path, types = {} }) => (
       <div style={{ background }}>
         <Group
-          tree={tree.tree || tree}
+          tree={
+            path ? treeUtils.lookup(path, tree.tree || tree) : tree.tree || tree
```

Comment:
Let's instead rely on a `getNode` function being passed in, which is part of contexture-client. In the unit tests we should mock the api like you did. While in theory, this lib doesn't need contexture-client, it does still need the top level things like dispatch or it wouldn't be useful so we should probably document the interface expectations better